scripts/util.sh

```diff
@@ -3,6 +3,11 @@
 #
 # A set of helper functions to use in other scripts.
 
+if [ -z "$BASH" ]; then
+    echo "ERROR: This script currently only works using a bash shell." >&2
+    return 1
+fi
```

Comment:
Checks for just bash for now since I used a `local ` var below which `sh` doesn't support.